tests/test_core.py

```diff
@@ -5144,12 +5144,14 @@ def test_fs_append(self):
   def test_fs_mmap(self):
     self.uses_es6 = True
     orig_compiler_opts = self.emcc_args[:]
-    for fs in ['MEMFS', 'NODEFS']:
+    for fs in ['MEMFS', 'NODEFS', 'NODERAWFS']:
       src = path_from_root('tests', 'fs', 'test_mmap.c')
       out = path_from_root('tests', 'fs', 'test_mmap.out')
       self.emcc_args = orig_compiler_opts + ['-D' + fs]
       if fs == 'NODEFS':
         self.emcc_args += ['-lnodefs.js']
+      if fs == 'NODERAWFS':
+        self.emcc_args += ['-sNODERAWFS=1']
```

Comment:
Does `-lnoderawfs.js` work?  (like the line two lines above?)

Reply:
That works too, but in that case also `-lnodefs.js` needs to be linked.

(Perhaps we could just synonym `-sNODERAWFS=1` with `-lnoderawfs.js`? See for e.g. https://github.com/kleisauke/emscripten/commit/b3607b9d591044e00a1be71ad10641b3bb285da3)

Reply:
I included `-lnodefs.js` and `-lnoderawfs.js` with commit https://github.com/kleisauke/emscripten/commit/244e4786237d284ac3971949fdbfbf97fc904f12, but perhaps this (i.e. `['-sNODERAWFS=1']` -> `['-lnodefs.js', '-lnoderawfs.js']`) should be done throughout the whole test suite? Let me know if you want a separate PR for this.